Multisig/Cross-layer/FirebaseRemoteConfig.swift

```diff
@@ -25,6 +25,10 @@ class FirebaseRemoteConfig {
         remoteConfig = RemoteConfig.remoteConfig()
         let settings = RemoteConfigSettings()
         remoteConfig.configSettings = settings
+        // we have this codition to make it faster to test on staging
+        if Bundle.main.isStaging {
```

Comment:
Sorry but this extension on Bundle.main is not needed because we already have this thing implemented:

`App.configuration.environment != .production`

I would keep it as "not production" so that it would work both in dev and staging.